src/include/binder/copy/bound_copy_from.h

```diff
@@ -11,32 +11,55 @@
 namespace kuzu {
 namespace binder {
 
+struct BoundCopyFromInfo {
+    std::unique_ptr<common::CopyDescription> copyDesc;
+    catalog::TableSchema* tableSchema;
+    expression_vector columnExpressions;
+    std::shared_ptr<Expression> offsetExpression;
+    // `boundOffsetExpression` and `nbrOffsetExpression` are for rel tables only.
+    std::shared_ptr<Expression> boundOffsetExpression;
+    std::shared_ptr<Expression> nbrOffsetExpression;
+
+    bool preservingOrder_;
```

Comment:
Why do we need to add a '_' after the variable name? For getters, you can use `isPreservingOrder`